pkg/process/config/config.go

```diff
@@ -171,6 +171,8 @@ func NewDefaultTransport() *http.Transport {
 
 // NewDefaultAgentConfig returns an AgentConfig with defaults initialized
 func NewDefaultAgentConfig(canAccessContainers bool) *AgentConfig {
+	initRuntimeSettings()
```

Comment:
If you're adding `initRuntimeSettings()` here, should we remove if from `NewAgentConfig()`?

Reply:
Actually, now that I think about it we don't need this here since the client doesn't need to know about the runtime settings. I've removed it.